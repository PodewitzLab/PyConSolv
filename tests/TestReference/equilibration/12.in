minimization
&cntrl
	imin=1,ncyc=500,maxcyc=1000,ntr=1,cut=8.0,
	restraint_wt=20.0,restraintmask="!@H=&:1",
/
