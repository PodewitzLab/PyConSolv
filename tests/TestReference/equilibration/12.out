
          -------------------------------------------------------
          Amber 20 PMEMD                              2020
          -------------------------------------------------------

| PMEMD implementation of SANDER, Release 18

|  Compiled date/time: Sat Aug 20 11:06:42 2022
| Run on 09/01/2023 at 17:31:47

|   Executable path: pmemd.cuda
| Working directory: /home/rat/PYCONSOLV_DATA/v1/equilibration
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|   MDIN: 12.in                                                                 
|  MDOUT: 12.out                                                                
| INPCRD: 11.rst7                                                               
|   PARM: LIG_solv.prmtop                                                       
| RESTRT: 12.rst7                                                               
|   REFC: 11.rst7                                                               
|  MDVEL: mdvel                                                                 
|   MDEN: mden                                                                  
|  MDCRD: mdcrd                                                                 
| MDINFO: mdinfo                                                                
|  MDFRC: mdfrc                                                                 


 Here is the input file:

minimization                                                                   
&cntrl                                                                         
	imin=1,ncyc=500,maxcyc=1000,ntr=1,cut=8.0,                                    
	restraint_wt=20.0,restraintmask="!@H=&:1",                                    
/                                                                              


Note: ig = -1. Setting random seed to   487474 based on wallclock time in 
      microseconds.
| irandom = 1, using AMBER's internal random number generator (default).
 
|--------------------- INFORMATION ----------------------
| GPU (CUDA) Version of PMEMD in use: NVIDIA GPU IN USE.
|                    Version 18.0.0
| 
|                      03/25/2018
| 
| Implementation by:
|                    Ross C. Walker     (SDSC)
|                    Scott Le Grand     (nVIDIA)
| 
| Version 18 performance extensions by:
|                    David Cerutti     (Rutgers)
| 
| Precision model in use:
|      [SPFP] - Single Precision Forces, 64-bit Fixed Point
|               Accumulation. (Default)
| 
|--------------------------------------------------------
 
|----------------- CITATION INFORMATION -----------------
|
|    When publishing work that utilized the CUDA version
|    of AMBER, please cite the following in addition to
|    the regular AMBER citations:
|
|  - Romelia Salomon-Ferrer; Andreas W. Goetz; Duncan
|    Poole; Scott Le Grand; Ross C. Walker "Routine
|    microsecond molecular dynamics simulations with
|    AMBER - Part II: Particle Mesh Ewald", J. Chem.
|    Theory Comput., 2013, 9 (9), pp3878-3888,
|    DOI: 10.1021/ct400314y.
|
|  - Andreas W. Goetz; Mark J. Williamson; Dong Xu;
|    Duncan Poole; Scott Le Grand; Ross C. Walker
|    "Routine microsecond molecular dynamics simulations
|    with AMBER - Part I: Generalized Born", J. Chem.
|    Theory Comput., 2012, 8 (5), pp1542-1555.
|
|  - Scott Le Grand; Andreas W. Goetz; Ross C. Walker
|    "SPFP: Speed without compromise - a mixed precision
|    model for GPU accelerated molecular dynamics
|    simulations.", Comp. Phys. Comm., 2013, 184
|    pp374-380, DOI: 10.1016/j.cpc.2012.09.022
|
|--------------------------------------------------------
 
|------------------- GPU DEVICE INFO --------------------
|
|            CUDA_VISIBLE_DEVICES: not set
|   CUDA Capable Devices Detected:      1
|           CUDA Device ID in use:      0
|                CUDA Device Name: NVIDIA GeForce RTX 3070 Laptop GPU
|     CUDA Device Global Mem Size:   8191 MB
| CUDA Device Num Multiprocessors:     40
|           CUDA Device Core Freq:   1.56 GHz
|
|--------------------------------------------------------
 
 
| Conditional Compilation Defines Used:
| PUBFFT
| BINTRAJ
| CUDA
| EMIL

| Largest sphere to fit in unit cell has radius =    17.630

| New format PARM file being parsed.
| Version =    1.000 Date = 09/01/23 Time = 17:28:38

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals

| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

 getting box info from netcdf restart file
 NATOM  =    6251 NTYPES =       7 NBONH =    4686 MBONA  =    1564
 NTHETH =      12 MTHETA =       6 NPHIH =      18 MPHIA  =       0
 NHPARM =       0 NPARM  =       0 NNB   =   10967 NRES   =    1565
 NBONA  =    1564 NTHETA =       6 NPHIA =       0 NUMBND =       9
 NUMANG =      10 NPTRA  =       1 NATYP =       9 NPHB   =       0
 IFBOX  =       1 NMXRS  =       4 IFCAP =       0 NEXTRA =    1560
 NCOPY  =       0

| Coordinate Index Table dimensions:     8    8    7
| Direct force subcell size =     4.5656    4.5511    5.0372

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =      50, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   8.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  20.00000

Energy minimization:
     maxcyc  =    1000, ncyc    =     500, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

| Intermolecular bonds treatment:
|     no_intermolecular_bonds =       1

| Energy averages sample interval:
|     ene_avg_sampling =       1

Extra-points options:
     frameon =       1, chngmask=       1

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   36.525   Box Y =   36.408   Box Z =   35.260
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   40       NFFT2 =   40       NFFT3 =   36
     Cutoff=    8.000   Tol   =0.100E-04
     Ewald Coefficient =  0.34864
     Interpolation order =    4

    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  default_name                                                                    
     Mask !@H=&:1; matches     1 atoms
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  4686     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =  4686     0
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  1564     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =     4     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =    12     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =    12     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =     6     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =     6     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =    18     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =    18     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =     0     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =     0     0

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps

 
 Number of triangulated 3-point waters found:     1560

     Sum of charges from parm topology file =   0.00000756
     Forcing neutrality...

| Dynamic Memory, Types Used:
| Reals              294880
| Integers           295091

| Nonbonded Pairs Initial Allocation:     1044073

| GPU memory information (estimate):
| KB of GPU memory in use:     23067
| KB of CPU memory in use:     10467

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -2.8127E+04     7.2186E-02     7.6629E+00     PT          1

 BOND    =     5206.9352  ANGLE   =        0.7728  DIHED      =        0.0000
 VDWAALS =     5683.5923  EEL     =   -38805.8277  HBOND      =        0.0000
 1-4 VDW =       -0.1464  1-4 EEL =     -212.4445  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     50      -2.8127E+04     2.1133E-02     7.0258E-01     CL9        11

 BOND    =     5207.5476  ANGLE   =        0.7728  DIHED      =        0.0000
 VDWAALS =     5683.5748  EEL     =   -38807.0809  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0764  RESTRAINT  =        0.0145
 EAMBER  =   -28127.4074


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100      -2.8127E+04     2.1117E-02     7.0073E-01     CL9        11

 BOND    =     5207.5478  ANGLE   =        0.7728  DIHED      =        0.0000
 VDWAALS =     5683.5746  EEL     =   -38807.0723  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0760  RESTRAINT  =        0.0146
 EAMBER  =   -28127.3982


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    150      -2.8127E+04     2.1104E-02     6.9941E-01     CL9        11

 BOND    =     5207.5480  ANGLE   =        0.7728  DIHED      =        0.0000
 VDWAALS =     5683.5749  EEL     =   -38807.0831  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0757  RESTRAINT  =        0.0147
 EAMBER  =   -28127.4082


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -2.8127E+04     2.1094E-02     6.9827E-01     CL9        11

 BOND    =     5207.5482  ANGLE   =        0.7728  DIHED      =        0.0000
 VDWAALS =     5683.5748  EEL     =   -38807.0762  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0754  RESTRAINT  =        0.0147
 EAMBER  =   -28127.4010


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    250      -2.8127E+04     2.1083E-02     6.9697E-01     CL9        11

 BOND    =     5207.5484  ANGLE   =        0.7728  DIHED      =        0.0000
 VDWAALS =     5683.5748  EEL     =   -38807.0770  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0751  RESTRAINT  =        0.0148
 EAMBER  =   -28127.4013


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300      -2.8127E+04     2.1073E-02     6.9579E-01     CL9        11

 BOND    =     5207.5485  ANGLE   =        0.7728  DIHED      =        0.0000
 VDWAALS =     5683.5747  EEL     =   -38807.0949  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0748  RESTRAINT  =        0.0148
 EAMBER  =   -28127.4189


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    350      -2.8127E+04     2.1062E-02     6.9453E-01     CL9        11

 BOND    =     5207.5487  ANGLE   =        0.7729  DIHED      =        0.0000
 VDWAALS =     5683.5750  EEL     =   -38807.0750  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0745  RESTRAINT  =        0.0149
 EAMBER  =   -28127.3981


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400      -2.8127E+04     2.1048E-02     6.9292E-01     CL9        11

 BOND    =     5207.5489  ANGLE   =        0.7729  DIHED      =        0.0000
 VDWAALS =     5683.5752  EEL     =   -38807.0850  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0741  RESTRAINT  =        0.0149
 EAMBER  =   -28127.4073


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    450      -2.8127E+04     2.1037E-02     6.9159E-01     CL9        11

 BOND    =     5207.5491  ANGLE   =        0.7729  DIHED      =        0.0000
 VDWAALS =     5683.5749  EEL     =   -38807.0871  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0738  RESTRAINT  =        0.0150
 EAMBER  =   -28127.4092


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -2.8127E+04     2.1028E-02     6.9058E-01     CL9        11

 BOND    =     5207.5493  ANGLE   =        0.7729  DIHED      =        0.0000
 VDWAALS =     5683.5752  EEL     =   -38807.0881  HBOND      =        0.0000
 1-4 VDW =       -0.1452  1-4 EEL =     -212.0735  RESTRAINT  =        0.0150
 EAMBER  =   -28127.4096


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    550      -2.8128E+04     3.0605E-02     1.5393E+00     O        1940

 BOND    =     5208.3251  ANGLE   =        0.7724  DIHED      =        0.0000
 VDWAALS =     5684.4695  EEL     =   -38809.5248  HBOND      =        0.0000
 1-4 VDW =       -0.1469  1-4 EEL =     -212.6008  RESTRAINT  =        0.2224
 EAMBER  =   -28128.7056


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600      -2.8129E+04     4.3346E-02     2.7517E+00     O        1940

 BOND    =     5208.3226  ANGLE   =        0.7807  DIHED      =        0.0000
 VDWAALS =     5685.2137  EEL     =   -38810.9750  HBOND      =        0.0000
 1-4 VDW =       -0.1454  1-4 EEL =     -212.1119  RESTRAINT  =        0.2105
 EAMBER  =   -28128.9154


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    650      -2.8129E+04     3.1031E-02     1.7485E+00     O        1940

 BOND    =     5208.3641  ANGLE   =        0.7836  DIHED      =        0.0000
 VDWAALS =     5685.2277  EEL     =   -38811.0309  HBOND      =        0.0000
 1-4 VDW =       -0.1454  1-4 EEL =     -212.1126  RESTRAINT  =        0.2106
 EAMBER  =   -28128.9135

     .... RESTARTED DUE TO LINMIN FAILURE ...


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    700      -2.8129E+04     2.8165E-02     1.9235E+00     H1       1941

 BOND    =     5208.5676  ANGLE   =        0.7977  DIHED      =        0.0000
 VDWAALS =     5685.2997  EEL     =   -38811.2945  HBOND      =        0.0000
 1-4 VDW =       -0.1454  1-4 EEL =     -212.1171  RESTRAINT  =        0.2111
 EAMBER  =   -28128.8920


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    750      -2.8129E+04     2.2400E-02     1.2692E+00     H2       1942

 BOND    =     5210.0890  ANGLE   =        0.7978  DIHED      =        0.0000
 VDWAALS =     5690.0753  EEL     =   -38818.0862  HBOND      =        0.0000
 1-4 VDW =       -0.1460  1-4 EEL =     -212.3419  RESTRAINT  =        0.1417
 EAMBER  =   -28129.6119


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800      -2.8130E+04     2.1505E-02     8.8493E-01     H1       1941

 BOND    =     5210.1808  ANGLE   =        0.8175  DIHED      =        0.0000
 VDWAALS =     5689.8643  EEL     =   -38818.4102  HBOND      =        0.0000
 1-4 VDW =       -0.1456  1-4 EEL =     -212.1974  RESTRAINT  =        0.1929
 EAMBER  =   -28129.8906


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    850      -2.8130E+04     2.0950E-02     8.4657E-01     H1       1941

 BOND    =     5210.1817  ANGLE   =        0.8172  DIHED      =        0.0000
 VDWAALS =     5689.8644  EEL     =   -38818.4103  HBOND      =        0.0000
 1-4 VDW =       -0.1456  1-4 EEL =     -212.1972  RESTRAINT  =        0.1929
 EAMBER  =   -28129.8898

     .... RESTARTED DUE TO LINMIN FAILURE ...


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    900      -2.8130E+04     1.4698E-02     4.5246E-01     CL9        11

 BOND    =     5210.1951  ANGLE   =        0.8116  DIHED      =        0.0000
 VDWAALS =     5689.8738  EEL     =   -38818.4195  HBOND      =        0.0000
 1-4 VDW =       -0.1456  1-4 EEL =     -212.1931  RESTRAINT  =        0.1930
 EAMBER  =   -28129.8777


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    950      -2.8130E+04     1.9614E-02     1.1112E+00     H1       1941

 BOND    =     5210.2999  ANGLE   =        0.8004  DIHED      =        0.0000
 VDWAALS =     5690.0012  EEL     =   -38818.6807  HBOND      =        0.0000
 1-4 VDW =       -0.1455  1-4 EEL =     -212.1757  RESTRAINT  =        0.1969
 EAMBER  =   -28129.9005


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -2.8130E+04     1.3092E-02     6.3714E-01     H1       1941

 BOND    =     5210.2617  ANGLE   =        0.7971  DIHED      =        0.0000
 VDWAALS =     5690.0108  EEL     =   -38818.6585  HBOND      =        0.0000
 1-4 VDW =       -0.1455  1-4 EEL =     -212.1707  RESTRAINT  =        0.1978
 EAMBER  =   -28129.9051


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -2.8130E+04     1.3092E-02     6.3714E-01     H1       1941

 BOND    =     5210.2617  ANGLE   =        0.7971  DIHED      =        0.0000
 VDWAALS =     5690.0108  EEL     =   -38818.6585  HBOND      =        0.0000
 1-4 VDW =       -0.1455  1-4 EEL =     -212.1707  RESTRAINT  =        0.1978
 EAMBER  =   -28129.9051
--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|  NonSetup CPU Time in Major Routines:
|
|     Routine           Sec        %
|     ------------------------------
|     Nonbond           0.51   63.33
|     Bond              0.00    0.00
|     Angle             0.00    0.00
|     Dihedral          0.00    0.00
|     Shake             0.00    0.00
|     Other             0.29   36.67
|     ------------------------------
|     Total             0.80

|  PME Nonbond Pairlist CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     Set Up Cit           0.00    0.00
|     Build List           0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  PME Direct Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     NonBonded Calc       0.00    0.00
|     Exclude Masked       0.00    0.00
|     Other                0.00    0.13
|     ---------------------------------
|     Total                0.00    0.13

|  PME Reciprocal Force CPU Time:
|
|     Routine              Sec        %
|     ---------------------------------
|     1D bspline           0.00    0.00
|     Grid Charges         0.00    0.00
|     Scalar Sum           0.00    0.00
|     Gradient Sum         0.00    0.00
|     FFT                  0.00    0.00
|     ---------------------------------
|     Total                0.00    0.00

|  Setup CPU time:            1.05 seconds
|  NonSetup CPU time:         0.80 seconds
|  Total CPU time:            1.85 seconds     0.00 hours

|  Setup wall time:           1    seconds
|  NonSetup wall time:        1    seconds
|  Total wall time:           2    seconds     0.00 hours
